cmd/nodecmd/loadtestStop.go

```diff
@@ -129,6 +160,12 @@ func stopLoadTest(_ *cobra.Command, args []string) error {
 			if err = destroyNode(existingSeparateInstance, clusterName, loadTestName, loadTestEc2SvcMap[separateHostRegion], nil); err != nil {
 				return err
 			}
+			for _, sg := range filteredSGList {
+				if err = deleteMonitoringSecurityGroupRule(ec2SvcMap[sg.region], loadTestNodeConfig.PublicIPs[0], sg.securityGroup); err != nil {
```

Comment:
can we rename the function name? it seems it is more general than just monitoring sg (load test in this case). also change naming inside the function, like the param monirotingHostPublicIP